### DISCUSSION TITLE
How can I click one pic and all pics from the page can be viewed

### DISCUSSION THREAD

**Discussion by aoyama798** [fengyuanchen/viewerjs#629] [Q&A]
Use for anki tamplate
If I ues DOM I can only get one pic or a list, but I want to get the whole gallary. What should I do to make this with a loop statement and bind the click event to each image?

var images = document.getElementsByTagName("img");
    for (let i = 0; i < images.length; i++) {
    images[i].addEventListener('click', function() { 
    
}

**Comment by fengyuanchen** [comment]
Just init a Viewer instance on the parent container of all the images.
